server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java

```diff
@@ -275,6 +325,9 @@ public ImmutableList<AnswerData> getSummaryDataOnlyActive() {
         String questionText = applicantQuestion.getQuestionText();
         String questionTextForScreenReader = applicantQuestion.getQuestionTextForScreenReader();
         String answerText = applicantQuestion.getQuestion().getAnswerString();
+        if (hiddenBlocks) {
+          answerText = "N/A";
```

Comment:
(question) Does this need to be localized?

Reply:
Not this doesn't. changes made.

Reply:
I think @gwendolyngoetz was asking if the "N/A" text needs to be translated into the user's language, not if it should not be a local variable.

The PDF probably should be translated, but other text in it is not so we shouldn't block on it. For example, the labels on each field: https://github.com/civiform/civiform/blob/cce0552baec4746f118423a64f6c37c09c7f2c5d/server/app/services/export/PdfExporter.java#L113C16-L113C31

Reply:
Thanks for providing more context!